src/api/cpp/cvc5.cpp

```diff
@@ -3309,31 +3308,6 @@ std::vector<Term> Term::getSequenceValue() const
   CVC5_API_TRY_CATCH_END;
 }
 
-bool Term::isUninterpretedValue() const
```

Comment:
I think we should keep these 2 methods.  They are analogous to other model getters for values, regardless of how values for uninterpreted sorts are represented.